styles.css

```diff
@@ -60,21 +64,221 @@ main {
   padding-top: 12px;
 }
 
-.page1 h2 {
+.sub-txt {
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
   padding-top: 12px;
 }
 
-.page1 ul {
+.social {
   display: flex;
   padding-top: 12px;
-  padding-bottom: 136px;
   list-style: none;
 }
 
-.page1 ul li {
+.social li {
   margin-right: 20px;
 }
+
+.works {
+  width: 100%;
+  margin-top: 114px;
+  padding: 0 16px;
+}
+
+.works1 {
+  width: 100%;
+  background: white;
+  border-radius: 16px;
+  padding: 16px;
+  margin-bottom: 88px;
+  border: #dfe1e6 solid 1px;
+}
```

Comment:
- [ ] As the project required, it is mandatory to use flexbox and grid. For each individual project of the works section, you should use CSS flexbox. Kindly use it to match the project requirements.

Reply:
@leonardodiasb 
I adjust `Flexbox` in sections, Obs 3
![image](https://user-images.githubusercontent.com/110146573/221338200-efc62eaa-127a-4377-b332-3d335faa5fb6.png)